app/json/options.json

```diff
@@ -19,7 +19,7 @@
       "postparagraph": ""
     },
     {
-      "preparagraph": "Provide a default value for `None`:",
+      "preparagraph": "Using getOrElse we can provide a default value ("No value") when the optional argument (`None`) does not exist:",
```

Comment:
I like the change, could you add backticks around `getOrElse` so it's shown in monospaced fonts? This way we stress the fact that is a method.